music21/pitch.py

```diff
@@ -134,6 +163,28 @@ def _sortModifiers():
 
 accidentalModifiersSorted = _sortModifiers()
 
+def isValidAccidentalName(name: str) -> bool:
+    # check against official names (e.g. 'double-flat')
+    if name in accidentalNameToModifier:
+        return True
+    # check against official modifiers (e.g. '--')
+    if name in accidentalNameToModifier.values():
+        return True
+    # check against alternate supported names (e.g. 'eses')
+    if name in alternateNameToAccidentalName:
+        return True
+    return False
+
+def makeAccidentalName(name: str) -> str | None:
+    if name in accidentalNameToModifier:
+        return name
+    if name in modifierToAccidentalName:
+        return modifierToAccidentalName[name]
+    if name in alternateNameToAccidentalName:
+        return alternateNameToAccidentalName[name]
+    return None
```

Comment:
could be better to raise an exception, then standardizeAccidentalName can always return a string and be used in string contexts.

Reply:
Done.